src/native/corehost/hostcommon/files.cmake

```diff
@@ -7,7 +7,7 @@ include_directories(${CMAKE_CURRENT_LIST_DIR}/../json)
 
 # CMake does not recommend using globbing since it messes with the freshness checks
 list(APPEND SOURCES
-    ${CMAKE_CURRENT_LIST_DIR}/../json_parser.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/../json/json_parser.cpp
```

Comment:
Was there a technical reason for moving this file into a subdirectory? I think it was fine where it was.

Reply:
I've moved it back.
json/ directory had only one subdirectory which is now moved, so it felt that it's the right place for json_parser to be or (otherwise there was no reason for json in corehost/json/rapidjson).